spring-boot/admin/src/main/resources/application.yml

```diff
@@ -75,6 +75,31 @@ spring:
     rest:
       base-path: /api
 
+
+  security:
+    oauth2:
+      client:
+        registration:
+         # resource:
+          #  userInfoUri: https://api.github.com/user
+          github:
+            client-id: e188a243cc57ef90f2b5
+            client-secret: a5630e2f873890bdc8366aac0155e8113ece0997
+            scope[0]: user:email
+            scope[1]: read:user
+        #    accessTokenUri: https://github.com/login/oauth/access_token
```

Comment:
Related to the lines that are commented out:

* Will they be required later? Then I would suggest to add a TODO comment with reference to an open issue.
* If it is not required anymore, please delete.

Reply:
I added a TODO as these comments might be important for coatrack#183